src/makeFileCopy.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function getDirectoryPath(filePath) {
+  const splitedPath = filePath.split('/');
+
+  const directoryPath = splitedPath.slice(0, splitedPath.length - 1);
+
+  return directoryPath.join('/');
+}
+
+function makeFileCopy(fileToCopy, newFilePath) {
+  const fileToCopyPath = path.join(__dirname, fileToCopy);
+  const pathOfCopiedFile = path.join(__dirname, newFilePath);
+
+  if (getDirectoryPath(fileToCopyPath) === getDirectoryPath(pathOfCopiedFile)) {
+    return;
+  }
```

Comment:
I am not sure u need getDirectoryPath function

Reply:
It is needed, if we change the name of the file, it will change that file path